package.json

```diff
@@ -19,8 +19,8 @@
     "lerna-version": "lerna version --no-git-tag-version --include-merged-tags --conventional-commits --no-push --force-publish",
     "lerna-publish": "lerna publish from-package --no-push --force-publish --no-verify-access --no-verify-registry",
     "lint": "yarn eslint && yarn stylelint && yarn prettier-check",
-    "prettier-check": "yarn prettier --check . --ignore-path .gitignore",
-    "prettier-write": "yarn prettier --write . --ignore-path .gitignore",
+    "prettier-check": "prettier --no-config --ignore-unknown --loglevel warn --check .",
+    "prettier-write": "prettier --no-config --ignore-unknown --loglevel warn --write .",
```

Comment:
👍 yeah, we don't need a `yarn` prefix since it natively runs them from `node_modules` as part of the runner path.